test/common.js

```diff
@@ -617,7 +617,9 @@ exports.WPT = {
   assert_false: (value, message) => assert.strictEqual(value, false, message),
   assert_throws: (code, func, desc) => {
     assert.throws(func, (err) => {
-      return typeof err === 'object' && 'name' in err && err.name === code.name;
+      return typeof err === 'object' &&
+             'name' in err &&
+             err.name.startsWith(code.name);
```

Comment:
I don't know about this one, I'm generally not a huge fan of `instanceof` checks due to unreliability across different V8 contexts, and it is quite possible that WPT's [testharness.js](https://github.com/w3c/web-platform-tests/blob/e1dcc277e0b0d1d51684c9e0bace2b253a7d1ac3/resources/testharness.js#L1184) uses the error name string check instead of `instanceof`.